appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/DraggableChildren.kt

```diff
@@ -82,6 +84,11 @@ fun <InteractionTarget : Any, ModelState : Any> DraggableAppyxComponent(
                     clipToBounds = clipToBounds
                 )
             }
+            .onPointerEvent {
```

Comment:
do you need this for puzzle15?

Reply:
For puzzle15, we can apply proposal from [this issue](https://github.com/bumble-tech/appyx/issues/494) and avoid carrying out gesture to next segment.

Reply:
yeah but in my mind they are not mutually exclusive

Reply:
They aren't, but let's do baby steps as gesture handling is core feature for this project.